app/controllers/search_controller.rb

```diff
@@ -4,7 +4,8 @@ class SearchController < ApplicationController
   layout 'search'
 
   def show
-    return if params[:search].nil? || params[:search].blank?
+    # See: http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F
```

Comment:
I think this comment is ambiguous. Following this pattern we can include URL with explanations to thousand of lines of code.